### PR TITLE
Updated contract metadata

### PR DESCRIPTION
**Pull request by achrafismyname**
related to https://github.com/curaOS/BSCS/pull/53

### CODE REVIEW

**Review by AshenI99 — APPROVED** [review]

File: as/cNFT/assembly/models/persistent_nft_contract_metadata.ts [hunk 1]
```diff
@@ -72,6 +60,18 @@ export class NFTContractExtra {
 
     /** Amount of royalty, in percentage, that is set on each minted token. */
     mint_royalty_amount: u32
+
+    /** Packages scripts of the contract if there is any */
+    packages_script: Map<string, Map<string, string>> | null
```
[the remark this answers is not in the document]
Comment (devey) [reply as/cNFT/assembly/models/persistent_nft_contract_metadata.ts:65]: Also, it's better to have typed `string` parameters in this case, so the structure is more explicit.

**Review by devey — APPROVED** [review]

File: as/cNFT/assembly/models/persistent_nft_contract_metadata.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (AshenI99) [reply as/cNFT/assembly/models/persistent_nft_contract_metadata.ts:65]: @devey  Seems like `null` with `Map` is valid. Maybe it seems like null cannot be used only with primitive data types.

File: tests/cNFT.js [hunk 2]
```diff
@@ -151,6 +147,16 @@ const CONTRACT_EXTRA = {
     mint_payee_id: 'jenny.test.near',
     mint_royalty_id: 'jenny.test.near',
     mint_royalty_amount: 10,
+    packages_script: {
```
Comment (devey) [inline tests/cNFT.js:150]: let's just abort this, let's keep it as it was, simply the string

**Review by devey — CHANGES_REQUESTED** [review]
I don't want to add too much overhead now, so let's keep `packages_script` as string.

**Review by devey — APPROVED** [review]
THANKS

**Pull request merged**